pkg/utils/utils.go

```diff
@@ -286,117 +230,117 @@ func DeleteDoubleWhiteSpace(slice []string) []string {
 	return result
 }
 
-// ExtractAnnotaions returns annotations of InvolvedObject for the given event
-func ExtractAnnotaions(obj *coreV1.Event) map[string]string {
+// ExtractAnnotationsFromEvent returns annotations of InvolvedObject for the given event
+func ExtractAnnotationsFromEvent(obj *coreV1.Event) map[string]string {
```

Comment:
We can get annotation directly from unstructured.Unstructured object

Reply:
This function is supposed to get the annotations from the `involvedObject` from the Event. If I don't convert it into `coreV1.Event`, would not be able to access the `involvedObject` filed of the `*unstructured.Unstructured`